net-p2p/syncthing/syncthing-0.14.12.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
 IUSE="tools"
 
+DEPEND="tools? ( >=dev-lang/go-1.7.1:= )"
```

Comment:
@khumarahn Ping :)
Since you added this flag. What do you suggest we call it? "relay" or something?